code/modules/implants/implantcase.dm

```diff
@@ -0,0 +1,103 @@
+/obj/item/weapon/implantcase
+	name = "Glass Case"
+	desc = "A hardened case containing an implant."
+	icon_state = "implantcase-blue"
+	item_state = "implantcase"
+	throw_speed = 1
+	throw_range = 5
+	w_class = W_CLASS_TINY
+	var/obj/item/weapon/implant/held_implant = null
+	var/implant_path = null
+	
+/obj/item/weapon/implantcase/New()
+	..()
+	src.held_implant = new implant_path(src)
```

Comment:
the many srcs in this file are unnecessary

Reply:
Copied from old implant code, will condense some of them.